driver/modern_bpf/programs/tail_called/events/syscall_dispatched_events/epoll_create1.bpf.c

```diff
@@ -0,0 +1,67 @@
+/*
+* Copyright (C) 2022 The Falco Authors.
+*
+* This file is dual licensed under either the MIT or GPL 2. See MIT.txt
+* or GPL2.txt for full copies of the license.
+ */
+
+#include <helpers/interfaces/fixed_size_event.h>
+
+/*=============================== ENTER EVENT ===========================*/
+
+SEC("tp_btf/sys_enter")
+int BPF_PROG(epoll_create1_e,
+	     struct pt_regs *regs,
+	     long id)
+{
+	struct ringbuf_struct ringbuf;
+	if(!ringbuf__reserve_space(&ringbuf, EPOLL_CREATE1_E_SIZE))
+	{
+		return 0;
+	}
+
+	ringbuf__store_event_header(&ringbuf, PPME_SYSCALL_EPOLL_CREATE1_E, EPOLL_CREATE1_E_SIZE);
+
+	/*=============================== COLLECT PARAMETERS  ===========================*/
+
+	/* Parameter 1: flags (type: PT_FLAGS32) */
+	s32 flags = (s32)extract__syscall_argument(regs, 0);
+	ringbuf__store_s32(&ringbuf, epoll_create1_flags_to_scap(flags));
```

Comment:
`epoll_create1_flags_to_scap` returns a `uint32_t` so we should use `ringbuf__store_u32`

```suggestion
	s32 flags = (s32)extract__syscall_argument(regs, 0);
	ringbuf__store_u32(&ringbuf, epoll_create1_flags_to_scap(flags));
```

Moreover `epoll_create1_flags_to_scap` should take as input a `int32_t` and not a `uint32_t`